### DISCUSSION TITLE
New uYou version is out

### DISCUSSION THREAD

**Discussion by tushark2** [arichornlover/uYouEnhanced#360] [General]
<a href="https://ibb.co/6DQjvX6"><img src="https://i.ibb.co/6DQjvX6/IMG-4759.png" alt="IMG-4759" border="0"></a>

**Comment by ego-lay-atman-bay** [comment]
What's the version number?

**Reply by arichornlover** [reply]
That’s uYou 3.0.3

**Comment by arichornlover** [comment]
> [![IMG-4759](https://camo.githubusercontent.com/1189a0827a8a878d475f90bc14c0ea5b751007a6bcc942ec0b61fe5f1f0d9ab5/68747470733a2f2f692e6962622e636f2f3644516a7658362f494d472d343735392e706e67)](https://ibb.co/6DQjvX6)

I know.. I haven’t updated uYou yet because I’m trying to implement something for uYouEnhanced.

**Comment by arichornlover** [comment]
https://github.com/arichornlover/uYouEnhanced/releases/tag/v19.06.2-3.0.3
